django/river/common/analyzer/analysis.py

```diff
@@ -15,7 +15,7 @@ def __init__(self):
         self.definition: Dict = None
         self.attributes: List[Attribute] = []
         self.filters: List[SqlFilter] = []
-        self.primary_key_column: SqlColumn = None
+        self.primary_key_column: Optional[SqlColumn] = None
```

Comment:
did you make this optional just to initialize it to None ?

Reply:
Yep, my editor was complaining about that